### ISSUE TITLE
Unable to Get Single MetaField

### ISSUE THREAD

**Issue by robodoginc** [nozzlegear/ShopifySharp#81]
When using `await service.GetAsync(metafieldId);` the returned data is null. If I use `await service.ListAsync(productId, "products");` it successfully returns all metafields related to that product. From that list, the MetaFieldId is the exact same as the one used in the GetAsync.

**Comment by oleksii-vynnychenko** [comment]
@robodoginc I just tried reading some product metafield with that method in my application and it's working. Can you provide source code or try to debug it yourself?
 
Also as additional debugging routine I personally use - is to make exact same calls in Postman (popular REST testing app) as you do with this library (essentially it's all that this library do - making API calls, but with very useful models).

**Comment by robodoginc** [comment]
@oleksii-vynnychenko Below is the source code that is failing for testing. 

```
var shopProductService = new ShopifyProductService(myShopifyUrl, shopAccessToken);
var shopMetaService = new ShopifyMetaFieldService(myShopifyUrl, shopAccessToken);

using (dbEntities db = new dbEntities())
{
    foreach (SaleItemShopifyProduct oSaleItemShopifyProduct in db.SaleItemShopifyProducts)
    {
        var metafields = await shopMetaService.ListAsync(oSaleItemShopifyProduct.Id, "products");

        foreach (SaleItemShopifyMetaField oSaleItemShopifyMetaField in oSaleItemShopifyProduct.SaleItemShopifyMetaFields)
        {
            long metaId = oSaleItemShopifyMetaField.Id;
            var sMeta1 = await shopMetaService.GetAsync(metaId);
            var sMeta = metafields.Where(q => q.Id == metaId).FirstOrDefault();
        }
    }
}
```

The code is looping through values stored in the database. When I load metafields (based upon the productId), it loads all related meta fields properly.

The metaId is the value returned from Shopify. The `.GetAsync` for `sMeta1 `is null, but the same Id in the loaded list, it finds and returns the object `sMeta `properly. I am using the same service for the MetaFields, therefore it should not be a connection issue, as other tasks through the same service (`ListAsync `and `UpdateAsync` are functioning properly, but `GetAsync` returns null).

**Comment by nozzlegear** [comment]
Thanks for reporting this! I've been super busy over the past couple of weeks and haven't had a chance to look into this yet, but I'll get on it as soon as possible.

**Comment by oleksii-vynnychenko** [comment]
@robodoginc Ah, sorry... you're right.
![image](https://cloud.githubusercontent.com/assets/20228387/19636324/4cc98df4-99d1-11e6-9789-f4aad5005ec5.png)

Problem, I guess is that response is not Metafield object, but `{"metafield": <metafield object>}` so Json.NET can't deserialize it, so it returns `null`.

**Comment by nozzlegear** [comment]
Turns out it was a problem [right here](https://github.com/nozzlegear/ShopifySharp/blob/1727a28f403ecfdbc87dc99961e48db6bb09d92a/ShopifySharp/Services/MetaField/ShopifyMetaFieldService.cs#L85), where the service was expecting `"metafields": {...}"` but should have been `"metafield"`. It wasn't caught by the tests because `.GetAsync()` was the one metafield test that somehow wasn't in the metafield test playlist.

Thanks again for opening this issue. The fix should be available on Nuget with ShopifySharp v3.1.1.

**Comment by darkstar74** [comment]
My apologies. I missed that when I submitted that service. Thanks for sorting it.
